src/main/java/com/aws/greengrass/builtin/services/telemetry/ComponentMetricIPCEventStreamAgent.java

```diff
@@ -39,6 +40,8 @@
 import static com.aws.greengrass.ipc.modules.ComponentMetricIPCService.PUT_COMPONENT_METRIC_SERVICE_NAME;
 
 public class ComponentMetricIPCEventStreamAgent {
+    private static final String STREAM_MANAGER_SERVICE_NAME = "aws.greengrass.StreamManager";
+    private static final List<String> AUTHORIZED_COMPONENTS = Collections.singletonList(STREAM_MANAGER_SERVICE_NAME);
```

Comment:
We need to tag this line with the SIM that we have created for revisiting defaultConfiguration behavior. I like the idea of keeping this a list, but do we expect its size to increase ?

Reply:
This is a common practice. We have done that in other places as well.

Reply:
+1 on adding a comment here